SDK/New Group/Models/SettingsObject.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  SettingsObject.swift
+//  iOS-SDK
+//
+//  Created by Alberto Huerdo on 12/17/18.
```

Comment:
please either remove the "created by" comment or set the date format to "yyyy-mm-dd". The "mm/dd/yy" format is really ambiguous in a distributed team.

Reply:
Changed the dates format to match all files match files created by Balazs, now they look like this "2018. 02. 20."

Reply:
Changed the dates format to match all files match files created by Balazs, now they look like this "2018. 02. 20."